packages/xo-server/src/vhd-merge.js

```diff
@@ -288,7 +277,7 @@ export class Vhd {
     const buf = await this._read(0, VHD_FOOTER_SIZE + VHD_HEADER_SIZE)
 
     const sum = unpackField(fuFooter.fields.checksum, buf)
-    const sumToTest = checksumStruct(buf, fuFooter)
+    const sumToTest = checksumStruct(buf.slice(0, VHD_FOOTER_SIZE), fuFooter)
```

Comment:
That is not necessary for the code, but I added it when debugging because it took me quite some time to understand what was in front of me in the checksum function.

Reply:
I'll remove it then :smiley: